modules/developer-lightspeed/proc-customizing-the-chat-history-storage.adoc

```diff
@@ -15,33 +15,32 @@ If you configure {ls-short} to store chat history using PostgreSQL, prompts and
 ====
 
 .Procedure
-* When you are using {ls-short} on an Operator-installed {product-very-short} instance, in your {product-very-short} instance ConfigMap, update the `conversation-cache` field as shown in the following example:
-+
+Customize the chat history storage using any of the following methods:
+
+* Update your {product-very-short} config file.
+** If you are using {ls-short} on an Operator-installed {product-very-short} instance, update `conversation_cache` in your {product-very-short} ConfigMap.
+** If you are using {ls-short} on a Helm-installed {product-very-short} instance, update `conversation_cache` in your `values.yaml` file.
++ 
 [source,yaml]
 ----
  conversation_cache:
     type: postgres
     postgres:
       host: _<your_database_host>_
       port: _<your_database_port>_
-      dbname: _<your_database_name>_
+      db: _<your_database_name>_
```

Comment:
Preview shows that the spacing is all wrong

<img width="551" height="329" alt="Screenshot 2025-11-07 at 11 20 14 AM" src="https://github.com/user-attachments/assets/a93f414b-a70c-4cbc-808b-8c9abea18262" />

Its all in one nested level

```yaml
conversation_cache:
  type: "postgres"
  postgres:
    host: "localhost"
    port: 5432
    db: "lightspeed_cache"
    user: "cache_user"
    password: "password"
    ssl_mode:
    ca_cert_path: 
```

Reply:
also no `ssl_mode` or `ca_cert_path` can be configured for now. IIRC